sources/packages/backend/libs/integrations/src/services/disbursement-schedule/e-cert-generation.service.ts

```diff
@@ -322,4 +323,25 @@ export class ECertGenerationService {
       },
     });
   }
+
+  /**
+   * Get the lifetime maximums of CSLP and is used in the e-Cert generation.
+   * @param assessmentId disbursement assessment id.
+   * @param entityManager used to execute the commands in the same transaction.
+   * @returns lifetime maximums of CSLP.
+   */
+  async getCSLPLifetimeMaximums(
```

Comment:
Since this method is dedicated to get a single value from the JSON we can follow a similar approach as below.
https://github.com/bcgov/SIMS/blob/83ea826cde4cf36acf786ac71780fcac4e272bf3/sources/packages/backend/apps/queue-consumers/src/services/student-assessment/student-assessment.service.ts#L52-L64